server/compose/service/record.go

```diff
@@ -714,6 +719,133 @@ func (svc record) Bulk(ctx context.Context, skipFailed bool, oo ...*types.Record
 	}
 }
 
+func (svc record) UpdateAll(ctx context.Context, namespaceID uint64, moduleID uint64, values types.RecordValueSet, recordIds ...uint64) (err error) {
```

Comment:
`recordIds` is later refered as `excludeIds` inside `processUpdateAll`. It'll throw people off